Debug potential issue with Mac scale/px -> dip conversion

sgtcoolguy opened the pull request:
**JIRA:** https://jira.appcelerator.org/browse/[TICKET]

Provide a clear PR title prefixed with `[TICKET]`

**Optional Description:**

sgtcoolguy committed:
test: shadow image must be compared by density due to point based shadow values

sgtcoolguy committed:
refactor(ios): use dot notation for properties

build commented on tests/Resources/ti.ui.view.test.js:
```diff
@@ -14,7 +14,7 @@ const utilities = require('./utilities/utilities');
 const isCI = Ti.App.Properties.getBool('isCI', false);
 
-describe('Titanium.UI.View', function () {
+describe.only('Titanium.UI.View', function () {
```
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.view.test.js;
  Line: 17;
-->

- :warning: tests/Resources/ti.ui.view.test.js line 17 – Unexpected exclusive mocha test. (mocha/no-exclusive-tests)

build commented:
<!--
  0 failure: 
  2 warning:  There is no linke..., :mag: Can't find ...
  1 messages
  
  DangerID: danger-id-Danger;
-->


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>There is no linked JIRA ticket in the PR body. Please include the URL of the relevant JIRA ticket. If you need to, you may file a ticket on <a href="https://jira.appcelerator.org/secure/CreateIssue!default.jspa">JIRA</a></td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  :mag: Can't find junit reports at `./junit.*.xml`, skipping generating JUnit Report.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>:fist: The commits in this PR match our conventions! Feel free to Rebase and Merge this PR when ready.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e44e65154c5d22118f0599c5962c27cb90889bbb
</p>

sgtcoolguy committed:
build(ios): clean ios build scripts, try to handle already extracted module zips with symlinks

- handle bad extractions of modules when checking initial hash by wiping dir and re-extracting

sgtcoolguy committed:
test(android): skip broken test on Android
